Makefile

```diff
@@ -230,8 +230,11 @@ endif
 ifndef CRC_BINARY
 	CRC_BINARY = --crc-binary=$(GOPATH)/bin
 endif
+ifndef VERSION_TO_TEST
+	VERSION_TO_TEST = --crc-version=$(CRC_VERSION)
```

Comment:
should be `VERSION_TO_TEST = --crc-version=$(CRC_VERSION)-$(COMMIT_SHA)` so we are sure that same commit hash is part of version of the binary we are testing.

Reply:
Added the `COMMIT_SHA` to `--crc-version`. So default behavior now includes commit check. 

Originally, it was only intended to tell apart a binary from the past release on the host from the current one, and similar "big" issues. That's why the check was only rough.